test/suites/serverconfig.sh

```diff
@@ -63,6 +63,10 @@ _server_config_storage() {
   lxc config set storage.backups_volume "${pool}/backups"
   lxc config set storage.images_volume "${pool}/images"
 
+  # Validate the old location is really gone
+  [ ! -e "${LXD_DIR}/backups/" ]
```

Comment:
should we lose the trailing slashes on these? Not sure if this is done intentionally?